src/main/webapp/individuals.jsp

```diff
@@ -1384,6 +1382,50 @@ if (sharky.getNames() != null) {
       }
       %>
 
```

Comment:
i have a suggestion to make this much simpler. i am not sure why the code inside this jsp (_not_ your code) is loading `getMarkedIndividual()` _so many times_!  its pretty bad form.

so i will suggest rather you set an `int` value at the top (around line 186) e.g. `int encCount = 0` ... then the first time `myEncs.size()` is calculated -- use that value.

then all of this code you added can be reduced to your `<div>....</div>` code, but use `encCount` in there.